client/scss/elements/_forms.scss

```diff
@@ -223,6 +224,8 @@ input[type='checkbox']:checked:before {
   // stylelint-disable-next-line property-disallowed-list
   left: 2px;
   inset-inline-start: 2px;
+  filter:contrast(0);
+  fill:currentColor;
```

Comment:
This looks like prettier has not run - remember to run `npm run format` after changing frontend code.

Reply:
Ok. I will keep this in mind. Thank you for poiting it out :)